### PR TITLE
tools: correct the description of Allowed options in osdomap tool

### PR DESCRIPTION
**Pull request by xiaomanh**
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by batrick** [comment]
Please prefix the title of your commit message with the subcomponent you are changing. See also [1]. This is required for your PR to be merged.

[1] https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

**Comment by xiaomanh** [comment]
hi @oritwas , @mattbenjamin do you mind take a review? thanks

**Comment by xiaomanh** [comment]
hi,@cbodley do you mind take a review? thanks

File: src/tools/ceph_osdomap_tool.cc [hunk 1]
```diff
@@ -31,7 +31,7 @@ int main(int argc, char **argv) {
   desc.add_options()
     ("help", "produce help message")
     ("omap-path", po::value<string>(&store_path),
-     "path to mon directory, mandatory (current/omap usually)")
+     "path to osd directory, mandatory (current/omap usually)")
```
Comment (tchaikov) [inline src/tools/ceph_osdomap_tool.cc]: i'd suggest s/mon/omap/ instead.

**Comment by tchaikov** [comment]
@xiaomanh ping?

File: src/tools/ceph_osdomap_tool.cc (hunk 1 above)
Comment (xiaomanh) [reply src/tools/ceph_osdomap_tool.cc]: Really thanks your comment, and I will check it again.

**Comment by zy751713126** [comment]
lgtm

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2018-09-10_07:18:46-rados-wip-kefu-testing-2018-09-10-1219-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2018-09-11_07:48:23-rados-wip-kefu-testing-2018-09-10-1219-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2018-09-11_09:51:05-rados-wip-kefu-testing-2018-09-10-1219-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2018-09-11_11:46:31-rados-wip-kefu-testing-2018-09-10-1219-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2018-09-11_12:12:46-rados-wip-kefu-testing-2018-09-10-1219-distro-basic-mira/

**Pull request merged**